contracts/near/eth-prover/src/test_data/storageProof.json

```diff
@@ -0,0 +1,25 @@
+    "f871808080808080a03f1587ebf71f19f47449b08f0960630ddbd60d045ea68ee7d89fb1c5682d215a8080808080a06e1618ec47442688101bd341cb8627739ecfd6fc1f793b69191dceeed18a19348080a05344081f911ad616b960a8f2841eb55f4936a97ec5dae614e726c8c0d51e0eff80",
+    "f8669d3e3757038b7fea6585ca2d0a3dacd72d84bf0c7b916f169cea0348681bb846f8440180a082bab1eb1ecd4b6dccd6d65b93603e6596205eb39ba1e32084c63dd883b17057a0932cddc50793da935ccf915651ad67f6b746e9936fcc5614f0ff492563782c75"
+  ],
+  "contract_address": "00763f30eEB0eEF506907e18f2a6ceC2DAb30Df8",
+  "expected_account_state": "f8440180a082bab1eb1ecd4b6dccd6d65b93603e6596205eb39ba1e32084c63dd883b17057a0932cddc50793da935ccf915651ad67f6b746e9936fcc5614f0ff492563782c75",
+  "storage_key_hash": "7bef74619139dab09d59cafa630682f8f30445738dcfb7147514aa27399ae2ee",
+  "storage_proof": [
+    "f90211a07c66fa00b691dc5c5a66e43831b9922316c8740c3c5235e7aa705cebefbd478ca0bbf8c0df9d3837792180786e217cbcbfb6cb6c1ee790ec0dba83fc05248f2627a036b93382543cd1737ddc32dbc8cd4326da47551be33e3ca75af3ea98583edd93a0c024e6190b7ce41e20f27b5cd512ebf50d6c60a1467f3cad8acd46da3a4d3861a04134b666e4a8e3e28701e33bed36680ae025cc5f89f8d29457975148338017c4a0ad773b24d54d681b3d712d913c0bf515407818845922616360b9b5a381cd6299a076515bac0f65924da357aba2f6c312472a9ef94d1c459f2cd0380b84a91d46aca0a3501be4fecb38f8b48131e0f8f63d6be379248ffb7d6f27c6fab26314fd8396a0324a3b32bf60329775f3383b6c98e99683df56a0933516032147008d544af343a029f967b2d7ed040cda92ab4905f2976636a7bb16ba850679eca942e02d5b3458a024bdf35e1d8fd46cdee76420d508c722d7f47825356620c266094f6c66c67f7ea0426a496a8633ec2771fb15f58f646bf689eda39af535bcb9b316e9e601d1a342a00d9cd9aa39cdd4a91bcd4feb8c73db017dce78ec82ba6f14287775567615f247a02f3d28876d69fd8945b3fdeff87186bcb97c236ccefa3b6182507bc6e504606fa03b4078e843b8927c29753dfc91a44c246cb22ea3673a3bad8dab7c8f6b8ce226a061983eaffadbdc7d6e7a0f9d9f5c1c64f983a4a6724aaa2126a0b6c03c984f3280",
+    "f90191a015203b1feedefc3d12d0e185733524b4e2e5e8c2bdaf7abff246a1fed7ac95e3a0493a8147104d998868825ea6deb1322aae76680d265471262e1767a23bdd1a7fa09b406baa1899b648205b0dd527c9bbde6ada6e3b4fc531a35bd2537d8db733f7a0426dd1071bf0b88c5742facc7fea20e28e765ab489cfdbdd5153e15ea3e2b32ca03edd49ea3795ac11d96e7290b12c1040c1e483c90462e720c7f1d16f873dc751a0132bf193ced5ce4fdb9c534386d84cb653c802dc21694d7dd78863921fea2acda09c60c4557046c9882148d3470e3fde97e88e5d35bd25e1a3cc94384e66a7af1080a0dea2418bd69d857116931f580a50c80761622d7488224a24b233bd4eaaa3c3b7a0bacb8d28242096ba6306d1c073fdeacd9bb5df804e3426c3c945f37836db68e680a08da1ba6ee52fb112c7f4707b78e039c8b52647ac5ce1122f7191356387dc169380a0f00a423bf08615d7c08955ea29573cd2a20d6d19d483ce4116333353290eae0ea06dd01d697beb2d96a5a41e803708b1d2cafd5992d80f7bf6d796e6aead4989ed8080",
+    "e2a020ef74619139dab09d59cafa630682f8f30445738dcfb7147514aa27399ae2ee01"
+  ],
+  "expected_storage_value": true,
```

Comment:
Please add a positive test where the "expected_storage_value": false,

Reply:
Added in [2d9d932](https://github.com/aurora-is-near/rainbow-bridge/pull/875/commits/2d9d932ef85338a2f0140945e4b0e70ebe04ff67)